versions.yaml

```diff
@@ -30,6 +30,7 @@ module-sets:
     version: v0.11.0
     modules:
       - go.opentelemetry.io/otel/log
+      - go.opentelemetry.io/otel/log/logtest
```

Comment:
Do we want this released with a semantic version? Should it be only referenceable by commit hash?

Reply:
I see the following differences.

If we release it with a semver tag,, we give some "promise" that we would like to stabilize it and that this module is package is "safe" to be used outside our repositories. 

If we do not release it with a semver tag, we give no guarantees on the stability and we can freely modify the API so that it fits our needs.

I have no strong opinion, but maybe it is indeed wiser and safer to only reference this module with commit hash. WDYT?

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/6465/commits/c194ffc0f72d272918e659851ba02849a924ba5b